app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id, dependent: :destroy
+  has_many :comments, foreign_key: :author_id, dependent: :destroy
+  has_many :likes, foreign_key: :author_id, dependent: :destroy
+
+  def three_recent_posts
+    posts.limit(3)
+  end
```

Comment:
- [ ] Kindly sort the recent post based on the `created_at` date by either adding a default scope as you did on the post model or using the ActiveRecord `order` directly on the method.

Reply:
Thank you for the feedback, @gemmen29. 

I am a bit confused by this comment because you acknowledged that I have sorted the posts based on the `created_at` date using  [`default_scope`](https://github.com/Timbar09/Blog_App/pull/2/files#diff-3baef9453c5d8b6c8962ce9ff9ec6b570b2e9cbad934e9f95b6d217ff1029b07R5), which means I do not need to sort the posts directly on the method as they are already sorted.

Please provide more clarity so that I can understand. Thanks again.

Reply:
Yes, I think you are right. I misunderstood the functionality of `default_scope` as it's the first time I encounter it. After a bit of searching, I found it's a bad practice to use as it causes some unexpected behavior. I suggest to you use ActiveRecord `order` directly in the method, Then ask for another review to avoid any misunderstood with the next reviewer.

Reply:
Thank you for the clarification. 🙏